.github/workflows/frontend-testing.yml

```diff
@@ -40,3 +40,5 @@ jobs:
           working-directory: ./ui
       - name: Test Debezium UI Frontend
         run: yarn run unit:test
+      - name: Format
```

Comment:
Hey wanted to know why we need this here.

Reply:
I think it's not required any longer as we are already using with husky.